playbooks/openstack/openshift-cluster/files/user-data

```diff
@@ -1,13 +0,0 @@
-#cloud-config
-disable_root: true
-
-system_info:
-  default_user:
-    name: openshift
-    sudo: ["ALL=(ALL) NOPASSWD: ALL"]
-
-write_files:
-  - path: /etc/sudoers.d/00-openshift-no-requiretty
-    permissions: 440
-    content: |
-      Defaults:openshift !requiretty
```

Comment:
@jannispl Thanks! I don't currently have an OpenStack env to test against, so I fully expected there to be something off.